code/lab/dialogs/render_options.cpp

```diff
@@ -257,6 +304,36 @@ void RenderOptions::open(Button* /*caller*/) {
 	dialogWindow->AddChild(bloom_sldr);
 	y += bloom_sldr->GetHeight() + 2;
 
+	auto exposure_sldr= new Slider("Exposure", 0, 8, 0, y + 2, set_exposure, dialogWindow->GetWidth());
+	exposure_sldr->SetSliderValue(lighting_profile::current_exposure());
+	dialogWindow->AddChild(exposure_sldr);
+	y += exposure_sldr->GetHeight() + 2;
+	auto ppcv = lighting_profile::lab_get_ppc();
```

Comment:
Readability: There should be a newline separating this one from the block above it, to make it easier to spot since the blocks below it all use `ppcv` to set their slider values.